src/treq/test/test_response.py

```diff
@@ -98,3 +99,18 @@ def test_history_notimplemented(self):
 
     if not skip_history:
         test_history_notimplemented.skip = "History supported."
+
+    def test_fail_for_status_ok(self):
+        original = FakeResponse(200, Headers(), body=[b''])
```

Comment:
If we want to make that change (I don't feel strongly about it either way) I'd rather see it in a separate PR.